web/cobrands/oxfordshire/layout.scss

```diff
@@ -276,9 +272,7 @@ h4.static-with-rule {
 
 .shadow-wrap {
     ul#key-tools {
-        -webkit-box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
-        -moz-box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
-        box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
+        @include box-shadow(0 0 1em 1em $oxfordshire_very_light_green);
```

Comment:
If we wanted a less hacky solution, and we're sure that FMS is running under Sass 3.2 or greater, we could let `@mixin box-shadow()` accept [a variable number of arguments](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_arguments), and pass the resulting list into `@experimental` where it should just get magically turned into a comma-separated value.

I think. I've never tried it before.